sys-kernel/coreos-firmware/coreos-firmware-99999999.ebuild

```diff
@@ -15,8 +15,11 @@ if [[ ${PV} == 99999999* ]]; then
 	EGIT_REPO_URI="git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git"
 	KEYWORDS=""
 else
-	SRC_URI="mirror://gentoo/linux-firmware-${PV}.tar.gz"
-	KEYWORDS="amd64 arm64"
+	GIT_COMMIT="2eefafb2e9dcbafdf4b83d8c43fcd6b75fd4ac78"
+	SRC_URI="https://git.kernel.org/cgit/linux/kernel/git/firmware/linux-firmware.git/snapshot/linux-firmware-${GIT_COMMIT}.tar.gz
```

Comment:
This still gives the source a different name than Gentoo uses.  It should be `linux-firmware-${PV}.tar.gz`.